### PR TITLE
Added RedirectPath Logic.

### PR DESCRIPTION
**Pull request by ericdrosado**

### CODE REVIEW

File: Sources/Server/Response.swift [hunk 1]
```diff
@@ -57,6 +57,12 @@ public class Response {
         let header = buildHeader(statusCode: status200, contentLength: body.utf8.count)
         return header + body
     }
+
+    private func handleRedirect(request: Request) -> String {
+        let body = prepareBody(body: "Hello World", method: request.method)
+        let header = "HTTP/1.1 302\r\nContent-Length: \(body.utf8.count)\r\nContent-type: text/html\r\nLocation: /\r\n\r\n"
```
Comment (cdemyanovich) [inline Sources/Server/Response.swift:63]: Why is the `Location` header blank?

Comment (ericdrosado) [reply Sources/Server/Response.swift:63]: There is a root path assigned that gets lost in the slashes. `Location: /`

Comment (cdemyanovich) [reply Sources/Server/Response.swift:63]: Ah. I see that now.

**Review by cdemyanovich — APPROVED** [review]

File: Sources/Server/Response.swift (hunk 1 above)
Comment (tommcgee) [inline Sources/Server/Response.swift:63]: same comment about pulling `\r\n` into a constant

**Review by tommcgee — APPROVED** [review]

**Pull request merged**